### PR TITLE
Move .pom file in gradle build task

### PR DESCRIPTION
**Pull request by mikaelm12**
We're now generating .pom files. Adding it to the .gitignore file at the Java client level

### CODE REVIEW

File: clients/java/signalr/.gitignore [hunk 1]
```diff
@@ -7,5 +7,6 @@ bin/
 out/
 *.class
 *.jar
+*.pom
```
Comment (anurse) [inline clients/java/signalr/.gitignore]: Maybe we can just ignore the whole `build` folder?

Comment (mikaelm12) [reply clients/java/signalr/.gitignore]: So we're already ignoring the entire build folder. The problem was that if you just run the gradle build task, the `signalr-client-0.1.0-preview1.pom` file would get created in the root of the java client. It was only getting moved by MSBuild. I updated the PR to have gradle move the .pom file to the build directory immediately after creating it, and then removed the move task that was in the MSBuild code.

Comment (BrennanConroy) [reply clients/java/signalr/.gitignore]: Can this be removed then?

File: clients/java/signalr/build.gradle [hunk 2]
```diff
@@ -45,4 +45,7 @@ task generatePOM {
             }
         }
     }.writeTo("signalr-client-0.1.0-preview1.pom")
-}
\ No newline at end of file
+
+    ant.move file: "signalr-client-0.1.0-preview1.pom",
```
Comment (BrennanConroy) [inline clients/java/signalr/build.gradle]: Are these versions going to not be hardcoded in the future?

File: clients/java/signalr/.gitignore (hunk 1 above)
Comment (anurse) [reply clients/java/signalr/.gitignore]: Ah, good. Sounds fine to me then. Thanks for clarifying!

**Pull request merged**